src/cli/main.c

```diff
@@ -35,6 +35,10 @@ static char *binary_name = NULL;
 static NcHashmap *g_commands = NULL;
 static bool explicit_help = false;
 
+#define FLAGS_USAGE "\nOptions:\n" \
+                    "  -p, --path        Set the base path for boot management operations.\n" \
```

Comment:
It seems a little confusing to have this in two places in the usage screen. I think updating the usage strings makes more sense to me. Prevents needing to do extra logic for future flags that may be split between options. Maybe could make a little macro to create the display string with short and long options even.